app/styles/_variables.scss

```diff
@@ -35,7 +35,6 @@ $color-white: #fff;
 $color-blue: #204762;
 $color-pure-blue: #00f;
 $color-hover: #e0ebf3;
-$color-light: #eee;
```

Comment:
This is lighter than the color that replaces it, which means it may cause a11y issues. Did you check vs. AXE tools?